custom_components/tuya_local/devices/doel_tiplus_litterbox.yaml

```diff
@@ -0,0 +1,249 @@
+name: Litter Box
+products:
+  - id: ebf44398a24b7b6be3vnml
+    name: DOEL Ti+
+primary_entity:
+  entity: sensor
+  name: Cat Weight
+  icon: "mdi:cat"
+  class: weight
+  dps:
+    - id: 6
+      type: integer
+      name: sensor
+      unit: kg
+      class: measurement
+      mapping:
+        - scale: 10
+    - id: 134
+      type: integer
+      name: sensor
+      unit: lb
+      class: measurement
+      mapping:
+        - scale: 10
+secondary_entities:
+  - entity: sensor
+    name: Visits
+    icon: "mdi:emoticon-poop"
+    dps:
+      - id: 7
+        type: integer
+        name: sensor
+        unit: visits
+  - entity: sensor
+    name: Visits Duration
```

Comment:
Usually this will be either a daily total, or last visit. The name should make that clear.